test/auto/injectorSpec.js

```diff
@@ -2,6 +2,37 @@
 
 /* globals support: false */
 
+describe('injector.modules', function() {
+    it('should expose the loaded module info on the instance injector', function() {
+      var test1 = angular.module('test1', ['test2']).info({ version: '1.1' });
+      var test2 = angular.module('test2', []).info({ version: '1.2' });
+      module('test1');
+      inject(['$injector', function($injector) {
```

Comment:
I don't remember anything like that (why would we)?
Maybe we have decided to make `ngMocks` minification-safe (because it makes it easier for some people to incorporate it in their workflow), but not our tests.